config/config_test.go

```diff
@@ -415,6 +416,7 @@ func TestFullConfig(t *testing.T) {
 	cmpStrings(t, "request_validation.ipv4_private_networks", cfg.RequestValidation.IPv4PrivateNetworks[0], "1.1.1.0/24")
 	cmpStrings(t, "request_validation.ipv6_private_networks", cfg.RequestValidation.IPv6PrivateNetworks[0], "1111::/16")
 	cmpStrings(t, "request_validation.ipv6_private_networks", cfg.RequestValidation.IPv6PrivateNetworks[1], "2222::/16")
+	cmpBools(t, "generate_bid_id", cfg.GenerateBidID, false)
```

Comment:
Would suggest setting this as `true` in the above `fullConfig` variable and asserting it here to `true` also as this test is making sure the config is read correctly and false is anyway the default

Reply:
Done, good point!